infrastructure/__main__.py

```
@@ -0,0 +1,2 @@
+import project
```

Ruff (0.8.2)

`project` imported but unused

Remove unused import: `project`

(F401)